src/__test__/Book.test.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import { Provider } from 'react-redux';
+import renderer from 'react-test-renderer';
+import Book from '../components/Book';
+import store from '../redux/store';
+import book from './bookMOck';
+
+it('renders correctly', () => {
+  const tree = renderer
+    .create(
+      <Provider store={store}>
+        <Book book={book} />
+      </Provider>,
+    )
+    .toJSON();
+  expect(tree).toMatchSnapshot();
```

Comment:
- [ ] Some of the tests are failing. Please fix that. :slightly_smiling_face: 
![Screenshot from 2021-10-14 14-24-37](https://user-images.githubusercontent.com/22434660/137286707-a3b44ab6-8fb0-4251-ba80-be1cf8f25215.png)